assets/src/edit-story/app/media/media3p/api/media3pApiProvider.js

```diff
@@ -55,7 +55,18 @@ function Media3pApiProvider({ children }) {
   function getUrls(m) {
     if (m.type.toLowerCase() === 'image') {
       return Object.fromEntries(
-        new Map(m.imageUrls.map((u) => [u.imageName, u.url]))
+        new Map(
+          m.imageUrls.map((u) => [
+            u.imageName,
+            {
+              file: m.name,
```

Comment:
Would it make sense to update the test for 'useMedia3pApi' to include image urls?